x-pack/dockerlogbeat/handlers.go

```diff
@@ -81,6 +103,28 @@ func stopLoggingHandler(pm *pipelinemanager.PipelineManager) func(w http.Respons
 	} // end func
 }
 
+func readLogHandler(pm *pipelinemanager.PipelineManager) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var logReq LogsRequest
+		err := json.NewDecoder(r.Body).Decode(&logReq)
+		if err != nil {
+			http.Error(w, errors.Wrap(err, "error decoding json request").Error(), http.StatusBadRequest)
+			return
+		}
+
+		pm.Logger.Infof("Got logging request for container %s\n", logReq.Info.ContainerName)
```

Comment:
hm, should this be 'debug'?

Makes me wonder if we should introduce some kind of audit logging